pkg/settings/cresettings/defaults.json

```diff
@@ -38,9 +35,6 @@
 		},
 		"HTTPTrigger": {
 			"RateLimit": "every30s:3",
-			"AuthRateLimit": "1rps:-1",
-			"IncomingPayloadSizeLimit": "10kb",
```

Comment:
move to global level.

Reply:
gave this a little more thought and I don't see the need for every product to have a workflow-level incomingPayloadSizeLimit. I've raised this in the [doc](https://docs.google.com/document/d/1NpJNhEi6OUuQ7EQ80kun28HLPSy2dLGaLPH6U4Va4-s/edit?disco=AAABqj49kgs) and can revisit this if needed